src/Clide.Addin/Clide.Addin.csproj

```
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net471</TargetFramework>
```

VSMac targets net472

```suggestion
    <TargetFramework>net472</TargetFramework>
```